kodilla-testing/src/test/java/com/kodilla/testing/library/BookDirectoryTestSuite.java

```diff
@@ -8,9 +8,9 @@
 
 public class BookDirectoryTestSuite {
 
-    private final static LibraryDatabase LIBRARY_DATABASE_MOCK = mock(LibraryDatabase.class);
-    private final static BookLibrary BOOK_LIBRARY = new BookLibrary(LIBRARY_DATABASE_MOCK);
-    private final static LibraryUser LIBRARY_USER_JAN = new LibraryUser("Jan", "Kowalczyk", "123345679");
+    private final LibraryDatabase libraryDatabaseMock = mock(LibraryDatabase.class);
+    private final BookLibrary bookLibrary = new BookLibrary(libraryDatabaseMock);
+    private final LibraryUser libraryUserJan = new LibraryUser("Jan", "Kowalczyk", "123345679");
```

Comment:
Do tego  daj osobna metodę, która zwróci takiego testowego uzytkownika, bo to brzydko wygląda :)